EXAMPLES.md

```diff
@@ -217,6 +214,50 @@ cart.LineItems.Get(firstProduct, selectedOptions);
 cart.LineItems.Delete(firstProduct, selectedOptions);
 ```
 
+### Native web view checkout
+
+After creating an instance of `Cart` and adding items to it, you can use the `CheckoutWithNativeWebView` method to 
+start a native modal overlay on top of your game with a web view containing the checkout for the cart.
+
+`CheckoutWithNativeWebView` which takes in 3 callback parameters:
+
+* `CheckoutDidSucceed` is called when the user has completed a checkout successfully.
+
+* `CheckoutDidCancel` is called when the user cancels out of the checkout.
+
+* `CheckoutDidFail` is called when an error was encountered during the web checkout. The callback will be passed an instance of `ShopifyError` describing the issue.
+
+```cs
+// Sample code for adding some product variants to your cart.
+var cart = ShopifyBuy.Client().Cart();
+var secondProductVariants = (List<ProductVariant>) products[1].variants();
```

Comment:
I wonder if we should do:
```
var secondProduct = products[1];
var secondProductVariants = (List<ProductVariant>) secondProduct.variants();
```

It's always better to be completely obvious in documentation.